modules/exploits/multi/handler.rb

```diff
@@ -53,6 +53,11 @@ def initialize(info = {})
   end
 
   def exploit
+    if datastore['PAYLOAD'].starts_with? "cmd/"
+        print_good "Run these commands on the target"
+        print_line generate_payload.raw
```

Comment:
This value may not always be safe to print.

Reply:
Can you think of any cmd payloads that wouldn't be safe or that would give non-ascii characters?

Reply:
A user can supply whatever they want as a custom value for the cmd string, base64 serialization and other mutation as well as shear quantity of data could make bumping the raw payload introduce artifacts that the console may not display well.